__tests__/app.test.js

```diff
@@ -136,37 +136,39 @@ describe('GET /api', () => {
   })
 
 
-  describe.skip("GET /api/articles/:article_id/comments", () => {
+  describe("GET /api/articles/:article_id/comments", () => {
     test("200: should get a comment by its id", () => {
       return request(app)
-        .get("/api/articles/1/comments")
+        .get("/api/articles/5/comments")
         .expect(200)
         .then(({body})=>{
 
-          const {article} = body
-            expect(article).toHaveProperty("article_id", 1);
-            expect(article).toHaveProperty("comment_id", expect.any(Number));
-            expect(article).toHaveProperty("body", expect.any(String));
-            expect(article).toHaveProperty("votes", expect.any(Number));
-            expect(article).toHaveProperty("author", expect.any(String));
-            expect(article).toHaveProperty("created_at", expect.any(String));
-            
+          const {comments} = body
+
+          comments.forEach((comment)=>{
+            expect(comment).toHaveProperty("comment_id", expect.any(Number));
+            expect(comment).toHaveProperty("body", expect.any(String));
+            expect(comment).toHaveProperty("votes", expect.any(Number));
+            expect(comment).toHaveProperty("author", expect.any(String));
+            expect(comment).toHaveProperty("created_at", expect.any(String))
+            expect(comment).toHaveProperty("article_id", expect.any(Number));
+          })
         })
   
     });
-    xtest("should return error with msg Not Found for request not in database", () => {
+    test("200: should return 200 for an empty array for request with no comments", () => {
       return request(app)
-        .get("/api/articles/99999999")
-        .expect(404)
+        .get("/api/articles/100/comments")
+        .expect(200)
         .then(({body})=>{
-          const {msg}= body
-          expect(msg).toBe("Not Found")
+          const {comments}= body
+          expect(comments).toEqual([])
```

Comment:
What you're going for here is great: responding with an empty array when an article has no comments makes total sense.

However, there is no `article_id` of `100` in the database, so it seems like we might have two slightly distinct circumstances:
- The article exists but has no comments (then we can respond like you have asserted here)
- The article with the specified `article_id` doesn't exist (what response would you expect in these circumstances?)